extensions/amp-bodymovin-animation/validator-amp-bodymovin-animation.protoascii

```diff
@@ -47,6 +47,10 @@ tags: {  # <amp-bodymovin-animation>
       allow_relative: false
     }
   }
+  attrs: {
+    name: "renderer"
+    value_regex: "svg|html"
```

Comment:
`value_casei` is now a repeated field which can be used instead of `value_regex`.

This becomes

```
  value_casei: "html"
  value_casei: "svg"
````

Reply:
Done!
Also added invalid tests cases to the validation test file.